front/app/components/FormBuilder/components/FormFields/FormField/index.tsx

```diff
@@ -203,7 +188,7 @@ export const FormField = ({
   };
 
   const onDelete = (fieldIndex: number) => {
-    if (builtInFieldKeys.includes(field.key)) {
+    if (!!field.code && field.input_type !== 'page') {
```

Comment:
Why do you need the `!!` before `field.code`. It creates a small amount of noise but it doesn't add anything to the condition, as far as I can tell.